packages/beachball/src/bump/updateRelatedChangeType.ts

```diff
@@ -22,15 +22,15 @@ export function updateRelatedChangeType(
 
   const disallowedChangeTypes = packageInfos[pkgName].options?.disallowedChangeTypes ?? [];
 
-  let depChangeType = getMaxChangeType('patch', dependentChangeTypes[pkgName], disallowedChangeTypes);
+  let depChangeType = getMaxChangeType(MinChangeType, dependentChangeTypes[pkgName], disallowedChangeTypes);
```

Comment:
The "dependencyChangeType" in change file generation should be "none" when the change type is "none"